src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  color: string;
+
+  constructor(
+    color: string,
+    public a: number,
```

Comment:
Let's avoid calling variables like `a, b, c, st, pt, bd, smthng` => a name should be meaningful. In this case, let's call them `sideA`, etc